demo/tree-view-demo.tsx

```diff
@@ -0,0 +1,45 @@
+import * as React from 'react';
+import { TreeView } from '../src';
+import { TreeItemData } from '../src/components/tree-view/tree-view';
+
+interface TreeViewDemoState {
```

Comment:
I have to say that this is being actively contested on the slack. :-) `interface` is ok, until proven otherwise.